pkg/kritis/review/validating_transport_test.go

```diff
@@ -50,14 +50,21 @@ func TestValidatingTransport(t *testing.T) {
 		ObjectMeta: metav1.ObjectMeta{Name: "test-attestor"},
 		Spec: v1beta1.AttestationAuthoritySpec{
 			PrivateKeySecretName: "test-success",
-			PublicKeyData:        base64.StdEncoding.EncodeToString([]byte(pub)),
+			PublicKeyList:        []string{base64.StdEncoding.EncodeToString([]byte(pub))},
+		},
+	}
+	validAuthWithTwoKeys := v1beta1.AttestationAuthority{
```

Comment:
Could you add some tests where the auth has two valid keys?  Mainly to exercise the disjunctive logic (need attestation by only one key).  Thanks!

Reply:
Done.